core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutput.java

```diff
@@ -84,6 +85,12 @@
     private final static int MAX_CAPACITY_MEGABYTES = 1000;
     private final static int MIN_CAPACITY_MEGABYTES = 1;
     private static final String MAX_TRANSMISSION_STORAGE_CAPACITY_NAME = "Channel.MaxTransmissionStorageCapacityInMB";
+    private static final ExceptionStats diskExceptionStats = new ExceptionStats(
+            TransmissionFileSystemOutput.class,
+            "Unable to send telemetry to the ingestion service," +
+                    " and unable to store telemetry locally, so telemetry will be discarded." +
+                    " See previous warning(s) for reason unable to send telemetry to the ingestion service." +
```

Comment:
I think we can avoid this line?
"See previous warning(s) for reason unable to send telemetry to the ingestion service"